### PR TITLE
New Design, Styles Additions and Web tracking

### PR DESCRIPTION
**Pull request by Patowhiz**
@michaelclapham @maxwellfundi I have created the base fonts that are similar to the ones in the wix website. I have demonstrated how these additions could be used in the `scs` files through the `home.component.scs` file. Upon your approval I can proceed with including the font styles to the other components.

With increase in the text sizes caused by the Roboto font, we will have to discuss more about styling of the website.

### CODE REVIEW

**Comment by michaelclapham** [comment]
Hey Patrick, I'd much prefer if we just used Google Fonts CDN to include Roboto. On Android Roboto is already installed, no point wasting people's bandwidth. Instructions on how to do this here: https://stackoverflow.com/questions/18178867/how-can-i-use-googles-roboto-font-on-a-website

**Comment by Patowhiz** [comment]
@michaelclapham are you positive that pointing to the remote CDN will work even when offline after installation.  That was my main concern, I happened to face that problem on another similar project.
If that's so I can use the CDNs, I agree it's more advantageous if it will never be downloaded by default.

**Comment by Patowhiz** [comment]
Great, just understood,  no offline capability needed here

**Comment by maxwellfundi** [comment]
@Patowhiz 	any more additions on this?

**Comment by Patowhiz** [comment]
@maxwellfundi yes, you may review this PR. 
@michaelclapham I'm still aware of your suggestion on using the CDN fonts rather than hosted one. Will work on that.

At the moment I have only worked on the home page, footer and partially on the header. I've been following the design guidelines shared by @michaelclapham . I'm yet to add some images and the slides. Will commit the page animations recommended after the current commits are reviewed.

File: src/app/web-analytics.service.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+declare let gtag: Function;
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WebAnalyticsService {
+
+  constructor() { }
+
+  public eventEmitter(
```
Comment (michaelclapham) [inline src/app/web-analytics.service.ts]: I'd prefer this to be emitAnalyticsEvent so that it's a verb to make usage clearer.

**Comment by maxwellfundi** [comment]
@Patowhiz  I quite like the new design for the homepage! It looks great. Just adding more stuff to meet Gaeleen design and linking buttons to the correct places. 
I wonder how the tracking is working, In what format is the data collected? Where is it  stored?

**Comment by Patowhiz** [comment]
@michaelclapham this PR builds fine in my machine. I checked the PR and the files have been uploaded. I don't know why netlify is unable to resolve the path to the font styles. However, I have changed the fonts url to point to the google CDN as suggested above, this should also fix the build error related to the fonts file paths.

sample build

**Comment by Patowhiz** [comment]
@michaelclapham @maxwellfundi  netlify builds without an error.  You can check on the new styling. I'm rebuilding the other components and enhancing the existing ones to conform with the proposed design guidelines. After that I'll add in animations and slides to make the website look and feel much more modern

**Comment by Patowhiz** [comment]
I haven't put much thought on reducing the size of the curves even though  as outlined by the design document they are optional, will do so after adding the actual contents and animations.

**Comment by Patowhiz** [comment]
@maxwellfundi yes. I'm still at the stage of making sure that we get it right in terms of how the base styling and effects will generally work. After that it should be easy to link to the different components.
The tracking sends data to google analytics, this data includes, pageviews and events that we can label the way we want. Then the analyst will use google analytics to get insights about the web visitors
I have built in the base code for this, awaiting to just listen for the  appropriate events. The next thing will be creating an account 
account for testing.

**Pull request merged**